python/pyspark/mllib/tests.py

```diff
@@ -75,22 +75,23 @@
     # No SciPy, but that's okay, we'll skip those tests
     pass
 
-ser = PickleSerializer()
```

Comment:
I can see why `sc` needs to be created on a per-test-case basis, but is it possible to leave `ser` as-is and keep it here?